packages/@lwc/engine-server/src/__tests__/fixtures/attribute-dynamic/expected.html

```diff
@@ -1,5 +1,11 @@
 <x-attribute-dynamic>
   <template shadowrootmode="open">
+    <div data-foo="foo">
+    </div>
+    <div class="foo">
+    </div>
+    <div style="color: red;">
+    </div>
     <div class="foo" style="color: red;" data-foo="foo">
     </div>
   </template>
```

Comment:
It might be interesting to run the `@lwc/engine-server` fixtures with the static optimization on vs off. We can do that in a separate PR, though.